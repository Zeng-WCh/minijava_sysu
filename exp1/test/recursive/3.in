5-2-9+9-5-9+3-3-2-4-7-9-5-8-8-4+6+5+8+7+2+7-3+1-3-7+8-7+3-5