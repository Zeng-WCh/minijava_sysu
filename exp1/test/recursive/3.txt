3-9+6+9-5+1-0+9+3-4-0-2+2+2-6+2+9+8-8+6+7+6+4+5-5-0+8-3-9+4