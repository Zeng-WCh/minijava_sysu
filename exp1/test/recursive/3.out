Input an infix expression and output its postfix notation:
52-9-9+5-9-3+3-2-4-7-9-5-8-8-4-6+5+8+7+2+7+3-1+3-7-8+7-3+5-
End of program.
